src/Analysis/Engine/Impl/ModuleTable.cs

```diff
@@ -416,7 +416,7 @@ private class InitializedModuleLoadState : ModuleLoadState {
                                     return false;
                                 }
 
-                                foreach (var location in type.Locations) {
+                                foreach (var location in type.Locations.WhereNotNull()) {
```

Comment:
Does this need a `MaybeEnumerate`, or is `type.Locations` known to be non-null at this point?